eng/pipelines/official/stages/publish.yml

```diff
@@ -1,5 +1,5 @@
 parameters:
-  PublishRidAgnosticPackagesFromPlatform: windows_x64
+  PublishRidAgnosticPackagesFromPlatform: browser_wasm
```

Comment:
`windows_x64` leg is not building the packages after #2133. We need to tell the publish process to pick-up the RID-neutral packages from `browser_wasm` leg.